backend/services/music_gen_service/music_utils.py

```
@@ -0,0 +1,78 @@
+def get_chord_midi_notes(chord_name: str, key: str) -> list[int]:
+    """
+    Get the MIDI note values for a given chord name and key.
+
+    Args:
+        chord_name: String of chord name (e.g. "C", "Cm", "Cmaj7", "C7", "C9", "C13")
+
+    Returns:
+        List of MIDI note values
+    """
+    chord = harmony.Chord(chord_name)
+    chord_scale = chord.getChordScale()
```

Ruff (0.8.2)

Local variable `chord_scale` is assigned to but never used

Remove assignment to unused variable `chord_scale`

(F841)